schema.sql

```diff
@@ -0,0 +1,84 @@
+CREATE database iCareNow;
+
+USE iCareNow;
+
+DROP TABLE IF EXISTS icn_role
+CREATE TABLE icn_role(
+	id	integer	not null identity primary key,
+	role_name varchar(20) not null,
+	)
+
+DROP TABLE IF EXISTS icn_user;
+CREATE TABLE icn_user(
+	id integer not null identity primary key,
+	email varchar(80) not null,
+	pwd varchar(100) not null,
+	account_status integer not null,
+)
+
+DROP TABLE IF EXISTS icn_user_role
+CREATE TABLE icn_user_role(
+	user_id integer not null,
+	role_id integer not null,
+	FOREIGN KEY (user_id) REFERENCES icn_user(id),
+	FOREIGN KEY (role_id) REFERENCES icn_role(id),
+	)
+
+DROP TABLE IF EXISTS icn_patient
+CREATE TABLE icn_patient(
+	id integer NOT NULL IDENTITY PRIMARY KEY,
+	id_user integer NOT NULL,
+	cnp varchar(14) not null,
+	first_name varchar(40) not null,
+	last_name varchar(40) not null,
+	insured bit not null,
+	FOREIGN KEY (id_user) REFERENCES icn_user(id),
+	)
+
+DROP TABLE IF EXISTS icn_doctor
+CREATE TABLE icn_doctor(
+	id integer NOT NULL IDENTITY PRIMARY KEY,
+	id_user integer not null,
+	first_name varchar(40) not null,
+	last_name varchar(40) not null,
+	FOREIGN KEY (id_user) REFERENCES icn_user(id),
+	)
+
+DROP TABLE IF EXISTS icn_specialization
+CREATE TABLE icn_specialization(
+	id integer NOT NULL IDENTITY PRIMARY KEY,
+	specialization_name varchar(40) not null,
+	)
+
+DROP TABLE IF EXISTS icn_doctor_specialization
+--make primary key combo
+CREATE TABLE icn_doctor_specialization(
+	doctor_id integer not null,
+	specialization_id integer not null,
+	FOREIGN KEY (specialization_id) REFERENCES icn_specialization(id),
+	FOREIGN KEY (doctor_id) REFERENCES icn_doctor(id),
+	PRIMARY KEY (doctor_id,specialization_id),
```

Comment:
pk should be above foreign key

Reply:
nu pricep

Reply:
Primary key trebuie pus deasupra la Foreign key ca linie, asta am inteles eu :D

Reply:
specialization_id integer not null,
        PRIMARY KEY (doctor_id,specialization_id),
        FOREIGN KEY (doctor_id) REFERENCES icn_doctor(id),
	FOREIGN KEY (specialization_id) REFERENCES icn_specialization(id),